src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int side1;
+    private int side2;
+
+    public Rectangle(String color, int side1, int side2) {
+        super(color);
+        this.side1 = side1;
+        this.side2 = side2;
+    }
+
+    public int getArea() {
+        return side1 * side2;
+    }
```

Comment:
The method name `getArea` implies that it is a getter, but it's actually calculating the area. It would be better to name this method `calculateArea` to reflect its purpose.